modules/microshift-otel-config.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-observability-service.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-otel-config_{context}"]
+= Configuring {microshift-short} Observability
+
+Configure the {microshift-short} service after it is installed to identify a valid endpoint.
+
+[IMPORTANT]
+====
+You must specify an OpenTelemetry Protocol (OTLP)-compatible endpoint for each configuration, such as Prometheus, before starting {microshift-short}. If an endpoint is not specified, the observability service will not begin.
+====
+
+.Procedure
+
+. Update the `/etc/microshift/observability/opentelemetry-collector.yaml` file to specify an OTLP-compatible endpoint with the following information. The endpoint must link to an IP address or host name, and port number of an OTLP service.
++
+.OTLP-compatible endpoint configuration
+[source,yaml]
+----
+exporters:
+  otlp:
+    sending_queue:
+      storage: file_storage
+    endpoint: ${env:K8S_NODE_NAME}:4317 <1>
+    tls:
+      insecure: true
+(...)
+service:
+(...)
+  telemetry:
+    metrics:
+      readers:
+        - periodic:
+            exporter:
+              otlp:
+                protocol: http/protobuf
+                endpoint: http://${env:K8S_NODE_NAME}:4318 <1>
+----  
+<1> Replace `${env:K8S_NODE_NAME}` with the IP address or host name of the remote back end. This IP address will resolve to the local node's host name. An unreachable endpoint is reported in the {microshift-short} service logs.    
```

Comment:
I changed it in https://github.com/openshift/microshift/pull/4903 so let's keep it consistent

```suggestion
    endpoint: ${env:OTEL_BACKEND}:4317 <1>
    tls:
      insecure: true
(...)
service:
(...)
  telemetry:
    metrics:
      readers:
        - periodic:
            exporter:
              otlp:
                protocol: http/protobuf
                endpoint: http://${env:OTEL_BACKEND}:4318 <1>
----  
<1> Replace `${env:OTEL_BACKEND}` with the IP address or host name of the remote back end. This IP address will resolve to the local node's host name. An unreachable endpoint is reported in the {microshift-short} service logs.    
```